draft-ietf-taps-interface.md

```diff
@@ -1367,8 +1367,8 @@ Provisioning Domain (PvD) or categories of PVDs it wants to
 consistent sets of network properties that might be more specific than network
 interfaces {{?RFC7556}}.
 
-As with interface instances and types, this property is a tuple of an (Enumerated)
-PvD identifier and a preference, and can either be implemented directly as such,
+As with interface instances and types, this property is a set of (Enumerated)
+PvD identifier and preference tuples. It can either be implemented directly as such,
```

Comment:
I find the tuple in the end very hard to read
```suggestion
As with interface instances and types, this property is a set of tuples of (Enumerated)
PvD identifier and preference. It can either be implemented directly as such,
```